### DISCUSSION TITLE
What unit tests generate coverage?

### DISCUSSION THREAD

**Discussion by mnicolaides** [coverlet-coverage/coverlet#1491] [General]
Coverlet has a great xml report of line/function/class coverage. My question is what unit test generated the coverage of a function/line? Is there a coverlet setting for that?

**Comment by daveMueller** [comment]
I'm not the super expert here but AFAIK coverlet doesn't track this infromation and thus it's not part of the generated reports. This was also once discussed here [https://github.com/coverlet-coverage/coverlet/issues/117#issuecomment-532555416](https://github.com/coverlet-coverage/coverlet/issues/117#issuecomment-532555416).

**Comment by Bertk** [comment]
You will find all information in a code coverage report created by [[ReportGenerator](https://github.com/danielpalme/ReportGenerator)]https://github.com/danielpalme/ReportGenerator).

coverlet coverage XML files are input for the report but a reference to tests name is not included.
